### PR TITLE
Update to the new theme custom properties

### PR DESCRIPTION
**Pull request by abdonrd**
Update to the new theme custom properties.
Reference: https://github.com/PolymerElements/polymer-starter-kit/issues/436#issuecomment-163551192

### CODE REVIEW

File: app/styles/app-theme.html [hunk 1]
```diff
@@ -24,16 +24,17 @@
   /* Application theme */
 
   :root {
-    --dark-primary-color: #303F9F;
-    --default-primary-color: #3F51B5;
```
Comment (robdodson) [inline app/styles/app-theme.html]: `--default-primary-color` is used here: https://github.com/PolymerElements/polymer-starter-kit/blob/master/app/styles/app-theme.html#L77

I just sent in a PR to update that spot with a mixin (which is also using `--default-primary-color`). 

I think this PR should update the above mentioned line to use `--primary-color` and then I can rebase and update my PR to also use `--primary-color`. Does that sound ok?

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by robdodson** [comment]
I only have the one comment above, @abdonrd can you update the PR to address it?

It looks like some of the colors vary ever so slightly from their original values but the page looks pretty similar to me. Here's an updated screenshot:

screen shot 2015-12-19 at 10 13 33 am

It may be so subtle that most folks won't notice.

File: app/styles/app-theme.html (hunk 1 above)
Comment (abdonrd) [inline app/styles/app-theme.html]: Ups! :sweat_smile:
Yeah! Sounds good to me!

**Comment by abdonrd** [comment]
Yes, @robdodson, the minor changes are:

`--primary-background-color` from `#c5cae9` to `#ffffff`.
`--secondary-text-color` from `#727272` to `#737373`.
`--disabled-text-color` from `#bdbdbd` to `#9b9b9b`.
`--divider-color` from `#B6B6B6` to `#dbdbdb`.

Now we have the same as Material Design spec. :smile:

**Comment by abdonrd** [comment]
By the way, PR updated! :dizzy:

**Comment by robdodson** [comment]
Change LGTM

**Comment by robdodson** [comment]
I'll wait for one of the other team members to :+1: then we can merge

**Comment by chuckh** [comment]
:+1:

**Comment by robdodson** [comment]
Thanks @abdonrd!

**Comment by abdonrd** [comment]
@robdodson, any news about this?

**Comment by robdodson** [comment]
Pining @notwaldorf for a review

**Comment by notwaldorf** [comment]
`--default-primary-color` is still used in some paper-elements because i haven't got around to go and fix any of them. so yes, if you change `var(--primary-color)` to teal and expect `paper-checkbox` to turn teal, you'll have a bad time.

The `paper-styles` change wasn't a breaking change, because `--default-primary-color` is still backward compatible, but I think this PR puts PSK ahead of all the paper-elements, which still depend on the `1.0.x` bits of `paper-styles` :(

**Comment by notwaldorf** [comment]
I thiiiiiiiink this is fine? Here's what I'm thinking
- you updated `paper-checkbox` 2 months ago. its dependency, `paper-styles/default-theme.html`, is from 2 months ago, so it’s `--default-primary-color` is old and hardcoded
- you update all your dependencies. there’s been no change to `paper-checkbox`, but when you update dependencies, `paper-styles/default-theme` gets updated to the future, because it has a new release (and in their `bower.json`, elements depend on `paper-styles#^1.0.0). in this future, `--default-primary-color` is aliased to `var(-primary-color)`
- so if you pull all the dependencies correctly, you can just update `var(--primary-color)` because all the elements will have the updated `default-theme.html`

ideally all the elements _should_ stop using `--default-primary-color` but i haven't got around to merging this.

**Comment by robdodson** [comment]
Thank you @notwaldorf :+1: :cake: :fireworks:

**Comment by abdonrd** [comment]
Nice! Thanks @notwaldorf!

**Pull request merged**